usr/src/man/man1m/zpool.1m

```diff
@@ -1607,6 +1612,29 @@ the pool, in addition to the pool-wide statistics.
 
 .sp
 .ne 2
+.mk
+.na
+\fB\fBzpool labelclear\fR [\fB-f\fR] \fIdevice\fR
+.ad
+.sp .6
+.RS 4n
+Removes ZFS label information from the specified device. The device must
+not be part of an active pool configuration.
```

Comment:
No, this can also be used on an exported pool if you specify the -f option.  Since the intent of the command is to result in a device without a label, if there is no label then the command treats it as a success.